catkin_tools/context.py

```diff
@@ -411,20 +446,20 @@ def summary(self, notes=[]):
                 "--init`.")]
         if not self.source_space_exists():
             summary_warnings += [clr(
-                "Source space `@{yf}{_Context__source_space_abs}@|` does not yet exist.")]
+                "Source space `@{yf}{__source_space_abs}@|` does not yet exist.")]
```

Comment:
Shouldn't this use the `self.source_space_abs` property? Why is the `_Context` prefix no longer needed, isn't it required due to the name mangling:

https://docs.python.org/2/tutorial/classes.html#private-variables-and-class-local-references